encoding/config.go

```diff
@@ -25,6 +26,8 @@ func MakeConfig(mb module.BasicManager) sdktestutil.TestEncodingConfig {
 	mb.RegisterLegacyAminoCodec(ec.Amino)
 	enccodec.RegisterInterfaces(ec.InterfaceRegistry)
 	mb.RegisterInterfaces(ec.InterfaceRegistry)
+	// This is needed for the EIP712 txs
+	legacytx.RegressionTestingAminoCodec = ec.Amino
```

Comment:
Why is this needed now?

Reply:
For the EIP712 transactions [we're using the `legacytx.StdSignBytes`](https://github.com/evmos/evmos/blob/9b0e8daa62505d0c5574e28c8e0eedfb3d764fb6/testutil/tx/eip712.go#L87).
In order to use this function successfully, [we need to set the `RegressionTestingAminoCodec`](https://github.com/cosmos/cosmos-sdk/blob/9814f684b9dd7e384064ca86876688c05e685e54/x/auth/migrations/legacytx/stdsign.go#L63-L66)